src/components/DisplayAccounts.jsx

```diff
@@ -51,18 +52,23 @@ export default function DisplayAccounts({ accounts, ...props }) {
           </rb.Accordion.Header>
           <rb.Accordion.Body>
             <Link to="/send" state={{ account }} className="btn btn-outline-dark">
-              Send
+              {t('current_wallet_advanced.account_button_send')}
             </Link>{' '}
             <Link to="/receive" state={{ account }} className="btn btn-outline-dark">
-              Receive
+              {t('current_wallet_advanced.account_button_receive')}
             </Link>
             {branches.map(({ balance, branch, entries }) => {
               const [type, derivation, xpub] = branch.split('\t')
               return (
                 <article key={derivation}>
                   <rb.Row className="mt-4 pe-3">
                     <rb.Col>
-                      <h6>{titleize(type)}</h6>
+                      {type === 'external addresses' && (
+                        <h6>{t('current_wallet_advanced.account_heading_external_addresses')}</h6>
+                      )}
+                      {type === 'internal addresses' && (
```

Comment:
Will `type` only ever have these two values? Should the raw `type` be displayed if `!['external addresses', 'internal addresses'].contians(type)`?

Reply:
Yeah right, I guess it could [theoretically be](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/1940764731d05cc6aca357b285a534d36b784bee/jmclient/jmclient/wallet_utils.py#L261) "Imported keys" as well. Added the fallback as you described. 👍